### ISSUE TITLE
Adding a black border to plotted colored points

### ISSUE THREAD

**Issue by pdarakjiank** [kevinblighe/PCAtools#35]
Hi Kevin, I am trying to add a black border to the colored points in biplot (usually, in R I use pch=21) but have not been able to. How do I change the shape of all points (they are colored by "group") to filled in the circles with a black border (keeping the colors that are already in place)? I need this rather than shape by group because it will help to distinguish multiple points when they are close together.

**Comment by kevinblighe** [comment]
Hi! _PCAtools_ is mainly based on _ggplot2_, so, you have to go by the _ggplot2_ shape codes. The same effect that you want can be availed of via the following parameters, I think:
```
shape = NUL
shapekey = NULL
```
There are sections in the vignette about these.

**Comment by pdarakjiank** [comment]
Yes, I got it. Thank you! Here is the code (in case anybody else needs to find out how to do it) that worked the way I needed it to:
`
biplot(p,
    lab=NULL,
    hline = 0,
    vline = 0,
    colby = 'Group',
    colkey = c('A.C'='yellow',
               'A.E'='purple',
               'H.C'='blue',
               'H.E'='darkgreen',
               'O.C'='darkblue',
               'O.E'='red',
               'P.C'='brown',
               'P.E'='black'),
    legendPosition = 'right',
    drawConnectors = F,
    gridlines.major = FALSE,
    gridlines.minor = FALSE,
    pointSize = 3.5) +
    geom_point(shape = 21,size = 3.5,colour = "black") +
  theme(text = element_text(size = 20),
        axis.text.x = element_text(size = 20),
        axis.text.y = element_text(size = 20),
        legend.key=element_blank(),
        panel.grid.major = element_blank(), 
        panel.grid.minor = element_blank(),
        panel.background = element_blank(),
        panel.border = element_rect(linetype = "solid", fill = NA))
`
